tests/accounts/test_csrf.py

```diff
@@ -0,0 +1,29 @@
+from django.conf import settings
+from django.test import Client
+from django.urls import reverse
+
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def _add_installed_apps(settings):
+    settings.LOGIN_URLS = [reverse("admin:login")]
+    settings.CSRF_FAILURE_VIEW = "maykin_common.accounts.csrf.csrf_failure"
+
+
+@pytest.mark.django_db
+def test_redirect_on_csrf_failure_if_logged_in(django_user_model):
+    user = django_user_model.objects.create_superuser(username="one")
+    client = Client(enforce_csrf_checks=True)
+    client.force_login(user)
+    response = client.post(reverse("admin:login"), {})
+
+    assert response.status_code == 302
+    assert response.url == settings.LOGIN_REDIRECT_URL  # type: ignore
+
+
+@pytest.mark.django_db
+def test_normal_csrf_failure_when_not_logged_in():
+    client = Client(enforce_csrf_checks=True)
+    response = client.post(reverse("admin:login"), {})
+    assert response.status_code == 403
```

Comment:
```suggestion
@pytest.mark.django_db
def test_normal_csrf_failure_when_not_logged_in():
    client = Client(enforce_csrf_checks=True)
    
    response = client.post(reverse("admin:login"), {})
    
    assert response.status_code == 403
```
here too - visual blocks :)

also can you add a comment to the `client` instantation that we *don't* use a fixture because by default csrf checks are not enforced. That should inform the next person that copy-pastes this test code that should have been using the `client` fixture instead.